templates/contact-us/form/index.html

```diff
@@ -37,13 +37,7 @@
   {% endwith %}
 
 {% elif product == 'pro' %}
-  {% with h1="Contact us about Ubuntu Pro",
-    intro_text="",
-    formid="1257",
-    lpId="2154",
-    returnURL="/contact-us/thank-you" %}
-    {% include "shared/_pro-contact-us-form.html" %}
-  {% endwith %}
+  {{ load_form("/contact-us/form?product=pro") | safe }}
```

Comment:
```suggestion
  {{ load_form("/contact-us/form?product=pro", isModal=True) | safe }}
```
Passing the param in the form is now being deprecated in place of passing it as a param

Reply:
Good catch! This form should be static so I'll just go ahead and `isModal` on the form gen file